public/_redirects

```diff
@@ -34,3 +34,10 @@ https://asyncapi.com/* https://www.asyncapi.com/:splat 301!
 /OSSRH-63280 https://github.com/asyncapi/java-asyncapi
 # Central Gradle repository verification
 /asyncapi-gradle-plugin https://plugins.gradle.org/plugin/com.asyncapi.asyncapi-gradle-plugin
+
+# TOOLS-REDIRECTION:START
+/parsers /tools/parsers 301!
+/modelina /tools/modelina 301!
+/generator /tools/generator 301!
+/github-actions /tools/github-actions 301!
```

Comment:
I have only problem with that. Should we make force redirect from old pages? @derberg How it will affect on currently SEO?

Reply:
Is there any other way? Better than broken links all over the place.

Reply:
Redirects is ok for me, but I wonder if it should be with 301 (force redirect) or 302 (temporary redirect) :)

Reply:
Was randomly reading next.js docs;  Found article on [redirection](https://nextjs.org/learn/seo/crawling-and-indexing/status-codes) related to SEO.

Reply:
Ok, 301 can be. If Fran accepted, I also 😄